docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -2,153 +2,150 @@
 - `eosio.msig` for multisig transactions
 - `eosio.system` for system level actions such as resource management
 
-Bootstrapping your local node is easy, once you have an active node running, you can bootstrap it with:
+Bootstrapping your local node is easy. Once you have an active node running, bootstrap it with:
 
 ```shell
 dune --bootstrap-system-full
 ```
 
-
 ## Account management
 
-You deploy contracts on top of accounts, and also use them to interact with your smart contracts. 
+You deploy contracts on top of accounts. Use accounts to interact with your smart contracts. 
 
-To create a new account:
+The following command creates a new account:
 
 ```shell
 dune --create-account <ACCOUNT_NAME>
 ```
 
-To get account info:
+The following command gets account info:
 
 ```shell
 dune -- cleos get account <ACCOUNT_NAME>
 ```
+## Smart contract development
 
-## Smart Contract Development
+Create a sample project to learn how to use DUNE to compile, deploy, and interact with smart contracts.
 
-Let's create a sample project so that we can learn how to compile, deploy, and interact with smart contracts using DUNE.
+### Create sample project
 
-Navigate to a directory you want to create a project in, and then run the following command:
+Navigate to a directory in which you want to create a project, then run the following command:
 
 ```shell
 dune --create-cmake-app hello .
 ```
-
-This will create a `hello` directory with a cmake style EOS smart contract project.
+This command creates a `hello` directory with a cmake style EOS smart contract project.
 
 Replace the contents of `src/hello.cpp` with the following code:
 
 ```cpp
-#include <eosio/eosio.hpp>
+#include <eosio/eosio.hpp> 
 using namespace eosio;
 
-CONTRACT hello : public contract {
-   public:
-      using contract::contract;
+CONTRACT hello : public contract {ompilinc
+ public:
+ using contract::contract;
 
       TABLE user_record {
          name user;
-         uint64_t primary_key() const { return user.value; }
+ uint64_t primary_key() const { return user.value; }
       };
-      typedef eosio::multi_index< name("users"), user_record> user_index;
+ typedef eosio::multi_index< name("users"), user_record> user_index;
 
       ACTION test( name user ) {
-         print_f("Hello World from %!\n", user);
+ print_f("Hello World from %!\n", user);
          user_index users( get_self(), get_self().value );
-         users.emplace( get_self(), [&]( auto& new_record ) {
-            new_record.user = user;
+ users.emplace( get_self(), [&]( auto& new_record ) {
+ new_record.user = user;
          });
       }
 };
 ```
-
 ### Compile the contract
 
 From the root of your project, run the following command to compile your contract:
 
 ```shell
 dune --cmake-build .
 ```
-You will see your contract being compiled. If there are any errors you will see them in the output.
+You see your contract compiling. If there are any errors, you see them in the output.
```

Comment:
Something about this sentence is weird, and hard to read. Perhaps it's the change in tense.

Reply:
The two sentences are passive voice. The rules are to take out the helping verb and to not use gerunds (-ing words).

Reply:
The reason for the change is to remove passive voice.

Developer writing is typically full of passive voice because of your exact statement - "They either will be putting the command in the future, or have already put the command in the past. It's very unlikely they will read this the same second they are clicking enter on the command."

Removing passive voice is tricky. The structure of "being compiled" infers and action going on in the background. "Is compiled" and "you will see them" is definitely passive and is understandable for developers. Whereas, for translation passive voice can get confusing.

I prefer the Past: "Your contract is now compiled. If there were any errors you see them in the output."